### DISCUSSION TITLE
I don't know what to write here and how (critical error on stable version)

### DISCUSSION THREAD

**Discussion by glory7sa** [hiddify/Hiddify-Manager#4593] [Bug باگ: please use issue]
1. Unable to install stable version of Hiddify Manager

**Comment by glory7sa** [comment]
I'll say right away that I'm not a programmer, I can only send logs. Tell me which ones?

**Reply by josephjul40** [reply]
wha to do, i cant get into my panel

**Reply by glory7sa** [reply]
You enter your panel through the terminal

**Comment by glory7sa** [comment]
![image](https://github.com/user-attachments/assets/387d4bab-2d4f-4a79-8b44-c5031939be10)

XD
